### PR TITLE
[10.0][MIG] payment killer migration

### PR DESCRIPTION
**Pull request by ovnicraft**
migration include move xml to new directories: view an security

### CODE REVIEW

**Comment by ovnicraft** [comment]
any review here ?

File: account_voucher_killer/view/invoice_view.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<openerp>
```
Comment (pedrobaeza) [inline account_voucher_killer/view/invoice_view.xml]: Use Odoo

File: account_voucher_killer/view/invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  <data>
```
Comment (pedrobaeza) [inline account_voucher_killer/view/invoice_view.xml]: Remove this unneeded tag

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by ovnicraft** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

File: account_voucher_killer/__manifest__.py [hunk 3]
```diff
@@ -19,16 +19,16 @@
 #
 ##############################################################################
 
-{'name': 'Accounting voucher killer',
- 'version': '8.0.1.0.0',
+{'name': 'Accounting Payment Access',
+ 'version': '10.0.1.0.0',
  'category': 'other',
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'website': 'http://www.camptocamp.com',
```
Comment (MeetKD) [inline account_voucher_killer/__manifest__.py]: Change `'website' : https://github.com/OCA/bank-payment`
May be you should define a meaningful category.

File: account_voucher_killer/view/invoice_view.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<odoo>
+    <record id="invoice_payment_group_customer_invoice" model="ir.ui.view">
+      <field name="name">Hide voucher in invoice</field>
```
Comment (MeetKD) [inline account_voucher_killer/view/invoice_view.xml]: Hide payment in...

**Review by MeetKD — CHANGES_REQUESTED** [review]

**Comment by ovnicraft** [comment]
@MeetKD done

**Comment by ovnicraft** [comment]
@MeetKD can you review it please ?

**Review by MeetKD — APPROVED** [review]

**Pull request merged**